www/docs/en/dev/cordova/storage/storage.md

```diff
@@ -166,6 +166,7 @@ IndexedDB is supported by the underlying WebView on the following Cordova platfo
 
 - Windows (with some limitations)
 - Android (4.4 and above)
+- iOS (14 and above)
```

Comment:
This is simply wrong.

Reply:
ok thanks for the clarification. Then you guys should merge it ASAP cause it produces so much confusion in terms of app architecture decisions